concordium-sdk/src/main/java/com/concordium/sdk/transactions/RawPayload.java

```diff
@@ -0,0 +1,39 @@
+package com.concordium.sdk.transactions;
+
+import com.concordium.sdk.types.UInt64;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+
+import java.util.Arrays;
+
+@ToString
+@EqualsAndHashCode(callSuper = false)
+public class RawPayload extends Payload {
+
+    /**
+     * The bytes of the raw payload.
+     */
+
+    @Getter
+    private final byte[] rawBytes;
+
+    public RawPayload(final byte[] bytes) {
+        this.rawBytes = bytes;
+    }
```

Comment:
Shouldn't you be checking here that there is at least 1 byte for the tag? And ideally that it's a valid tag?

Reply:
Good idea.